coderd/parameters.go

```diff
@@ -269,3 +275,31 @@ func (api *API) getWorkspaceOwnerData(
 		Groups:       groupNames,
 	}, nil
 }
+
+// parameterProvisionerVersionDiagnostic checks the version of the provisioner
+// used to create the template version. If the version is less than 1.5, it
+// returns a warning diagnostic. Only versions 1.5+ return the module & plan data
+// required.
+func parameterProvisionerVersionDiagnostic(tf database.TemplateVersionTerraformValue) hcl.Diagnostics {
+	missingMetaData := hcl.Diagnostic{
```

Comment:
```suggestion
	missingMetadata := hcl.Diagnostic{
```

"metadata" is one word